base/asyncmap.jl

```diff
@@ -62,7 +62,6 @@ function done(itr::AsyncCollector, state::AsyncCollectorState)
 end
 
 function next(itr::AsyncCollector, state::AsyncCollectorState)
-
```

Comment:
I assume that this change is related to the proposed rule in https://github.com/JuliaLang/julia/pull/15439
Please note that the proposed rule has not yet been accepted and that there are a number of dissenting viewpoints: 
- https://github.com/JuliaLang/julia/pull/15439#discussion_r55746138
- https://github.com/JuliaLang/julia/pull/15439#discussion_r55745011
- https://github.com/JuliaLang/julia/pull/15418#issuecomment-195064008
- https://github.com/JuliaLang/julia/pull/15418#issuecomment-195047162
- https://github.com/JuliaLang/julia/pull/15418#discussion_r57078286
- https://github.com/JuliaLang/julia/pull/15439#issuecomment-195117895